data/SugarBean.php

```diff
@@ -611,11 +611,11 @@ public static function createRelationshipMeta($key, $db, $tablename, $dictionary
      *
      * Internal Function, do not override.
      */
-    public static function get_union_related_list($parentbean = null, $order_by = "", $sort_order = '', $where = "",
-                                                  $row_offset = 0, $limit = -1, $max = -1, $show_deleted = 0, $subpanel_def)
+    public static function get_union_related_list($parentbean, $order_by = "", $sort_order = '', $where = "",
+                                                  $row_offset = 0, $limit = -1, $max = -1, $show_deleted = 0, $subpanel_def = null)
     {
-        if (empty($parentbean)) {
-            $GLOBALS['log']->fatal('empty parent bean');
+        if (is_null($subpanel_def)) {
```

Comment:
Don't think this is correct.... will subpanel_def be null?  Can't we both have parentbean = null and subpanel_def = null?